### PR TITLE
Add get_transactions to the Tx Service.

### PR DESCRIPTION
**Pull request by qnikst**

### CODE REVIEW

File: network-core/src/server/transaction.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 use crate::codes;
 use crate::error::Code as ErrorCode;
 
+use chain_core::property;
```
Comment (mzabaluev) [inline network-core/src/server/transaction.rs]: I tend to use `self` inside the brackets, but it's a matter of taste really.

Comment (mzabaluev) [reply network-core/src/server/transaction.rs]: And we could just use the `property::` path consistently in this module, for clarity.

File: network-core/src/server/transaction.rs [hunk 2]
```diff
@@ -12,6 +13,10 @@ use std::{error, fmt};
 /// Interface for the blockchain node service implementation responsible for
 /// validating and accepting transactions.
 pub trait TransactionService {
+
+    /// Transaction in the blockchain.
+    type Transaction: property::Transaction;
```
Comment (mzabaluev) [inline network-core/src/server/transaction.rs]: Alternatively, import `Transaction`, it does not conflict with associated type names.

**Comment by qnikst** [comment]
Thanks! fixed.

File: network-core/src/server/transaction.rs (hunk 2 above)
Comment (mzabaluev) [reply network-core/src/server/transaction.rs]: @qnikst TIL that you can also do `use property::Transaction as _;` go get the trait methods without importing the trait name.

Comment (NicolasDP) [reply network-core/src/server/transaction.rs]: Only since [rust 1.33](https://blog.rust-lang.org/2019/02/28/Rust-1.33.0.html#import-as-_)

**Pull request merged**